src/ZODB/Connection.py

```diff
@@ -873,9 +873,9 @@ def setstate(self, obj):
         except ConflictError:
             raise
         except:
-            self._log.exception("Couldn't load state for %s %s",
+            self._log.exception("Couldn't load state for %s %s %s",
                                 className(obj), oid_repr(oid),
-                                exc_info=sys.exc_info())
+                                sys.exc_info())
```

Comment:
log.exception(...) is the same as log.error(..., exc_info=True) is the same as log.error(..., exc_info=sys.exc_info()).

The correct fix would be to remove the exc_info argument from the log.exception() call here OR change the .exception() call to an .error() call.